lib/charms/layer/jenkins/configuration.py

```diff
@@ -17,12 +17,30 @@ def bootstrap(self):
         hookenv.log("Bootstrapping initial Jenkins configuration")
 
         config = hookenv.config()
-        context = {"master_executors": config["master-executors"]}
-        templating.render(
-            "jenkins-config.xml", paths.CONFIG_FILE, context,
-            owner="jenkins", group="nogroup")
 
-        hookenv.open_port(PORT)
+        if not -1 <= config["jnlp-port"] <= 65535:
+            err = "{} is not a valid setting for jnlp-port".format(
+                config["jnlp-port"]
+            )
+            hookenv.log(err)
+            hookenv.status_set("blocked", err)
+            return False
+        else:
```

Comment:
This else block creates unnecessary indentation. You can safely drop the "else", since in case the "if" branch is taken the method will return.